### PR TITLE
Translated usestate.md

### PR DESCRIPTION
**Pull request by Albert-Jokelin**
This is the fifth of the five files I translated.

### CODE REVIEW

File: beta/src/pages/apis/usestate.md [hunk 1]
```diff
@@ -4,38 +4,38 @@ title: useState
...
+- [संदर्भ](#reference)
   - [`useState(initialState)`](#usestate)
   - [`set` functions, like `setSomething(nextState)`](#setstate)
-- [Troubleshooting](#troubleshooting)
-  - [I’ve updated the state, but logging gives me the old value](#ive-updated-the-state-but-logging-gives-me-the-old-value)
-  - [I've updated the state, but the screen doesn't update](#ive-updated-the-state-but-the-screen-doesnt-update)
-  - [I'm getting an error: "Too many re-renders"](#im-getting-an-error-too-many-re-renders)
-  - [My initializer or updater function runs twice](#my-initializer-or-updater-function-runs-twice)
+- [ट्रबलशूटिंग](#troubleshooting)
+  - [मैंने state का वैल्यू अपडेट किया है मगर log करते समय मुझे पुराना वैल्यू मिलता है](#ive-updated-the-state-but-logging-gives-me-the-old-value)
```
Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:26]: `log` => `लोग`

Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:26]: `मिलता है` => `मिल रही है`

File: beta/src/pages/apis/usestate.md [hunk 2]
```diff
@@ -4,38 +4,38 @@ title: useState
...
+  - [state अपडेट कर लिया है लेकिन स्क्रीन अपडेट नही हो रहा](#ive-updated-the-state-but-the-screen-doesnt-update)
```
Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:27]: `हो रहा` => `हो रही`

File: beta/src/pages/apis/usestate.md [hunk 3]
```diff
@@ -4,38 +4,38 @@ title: useState
...
+  - [मुझे एक एरर मिल रहा है: "Too many re-renders"](#im-getting-an-error-too-many-री-renders)
```
Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:28]: `मिल रहा है` => `आ रहा है`

Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:28]: `#im-getting-an-error-too-many-री-renders` => `#im-getting-an-error-too-many-re-renders`

lets not translate anything related to links or reflinks

File: beta/src/pages/apis/usestate.md [hunk 4]
```diff
@@ -4,38 +4,38 @@ title: useState
...
+  - [मेरा reducer या एनिशिअलिज़ेर फंक्शन दो बार चलता है  ](#my-initializer-or-updater-function-runs-twice)
```
Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:29]: `reducer` => `रेडूयसर`

Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:29]: `एनिशिअलिज़ेर` => `इनिशिअलिज़ेर`

Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:29]: `है ` => `है`

remove extra space at the end

File: beta/src/pages/apis/usestate.md [hunk 5]
```diff
@@ -4,38 +4,38 @@ title: useState
...
   - [I'm trying to set state to a function, but it gets called instead](#im-trying-to-set-state-to-a-function-but-it-gets-called-instead)
```
Comment (arshadkazmi42) [inline beta/src/pages/apis/usestate.md:30]: You have missed this line, can you translate this line also.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

I have added review till line 30

Lets get these fixed and post that we can continue further with the review.

Also, you can refer our review process #23 to know more

**Review by Himanshu281 — APPROVED** [review]

File: beta/src/pages/apis/usestate.md [hunk 6]
```diff
@@ -4,38 +4,38 @@ title: useState
...
-  - [Avoiding recreating the initial state](#avoiding-recreating-the-initial-state)
-  - [Resetting state with a key](#resetting-state-with-a-key)
-  - [Storing information from previous renders](#storing-information-from-previous-renders)
-- [Reference](#reference)
+- [प्रयोग](#usage)
+  - [कौम्पोनॅन्ट में state ऐड करना](#adding-state-to-a-component)
+  - [पिचले state के आधार पर state अप्डेट करना](#updating-state-based-on-the-previous-state)
+  - [State में ऑब्जेक्ट्स और अर्रेस अपडेट करना](#updating-objects-and-arrays-in-state)
+  - [इनिशल state को फ़िर सृष्ट करने से बचना](#avoiding-recreating-the-initial-state)
+  - [ Key के सात state रिसेट करना](#resetting-state-with-a-key)
```
Comment (thoughtlessmind) [inline beta/src/pages/apis/usestate.md:20]:
```suggestion
  - [ Key के साथ state रिसेट करना](#resetting-state-with-a-key)
```

File: beta/src/pages/apis/usestate.md [hunk 7]
```diff
@@ -4,38 +4,38 @@ title: useState
...
+  - [पिचले रेंडर के जानकारी को स्टोर करना](#storing-information-from-previous-renders)
```
Comment (thoughtlessmind) [inline beta/src/pages/apis/usestate.md:21]:
```suggestion
  - [पिछले रेंडर के जानकारी को स्टोर करना](#storing-information-from-previous-renders)
```

File: beta/src/pages/apis/usestate.md (hunk 1 above)
Comment (thoughtlessmind) [reply beta/src/pages/apis/usestate.md:26]: then we've to change the word `पुराना` to `पुरानी`
```suggestion
  - [मैंने state का वैल्यू अपडेट किया है मगर log करते समय मुझे पुरानी वैल्यू मिल रही है](#ive-updated-the-state-but-logging-gives-me-the-old-value)
```

File: beta/src/pages/apis/usestate.md [hunk 8]
```diff
@@ -44,28 +44,29 @@ function MyComponent() {
...
+स्क्रीन को अप्डेट करने के लिए, `set` फ़ंक्शन को किसी और state के साथ बुलाओ|
 
 ```js [[2, 2, "setName"]]
 function handleClick() {
   setName('Robin');
 }
 ```
 
-React will store the next state, render your component again with the new values, and update the UI.
+React अगला state स्टोर कर देगा, नए वैल्यूस के सात कौम्पोनॅन्ट रेंडर करेगा और UI को अप्डेट करेगा|
```
Comment (thoughtlessmind) [inline beta/src/pages/apis/usestate.md:65]:
```suggestion
React अगला state स्टोर कर देगा, नए वैल्यूस के साथ कौम्पोनॅन्ट रेंडर करेगा और UI को अपडेट करेगा।
```

**Comment by thoughtlessmind** [comment]
hey @arshadkazmi42 
I think keeping the translation of the word `call` to `कॉल` only instead of `बुलाओ`. `कॉल` would make more sense in terms of calling function rather than `बुलाओ`.

I want to know your thoughts on this

**Comment by arshadkazmi42** [comment]
@Albert-Jokelin There are some conflicts in the file. Can. you resolve those or open a new PR. I will get this merged once that is resolved

**Comment by arshadkazmi42** [comment]
No response from author

**Pull request closed without merging**